pipeline/api/run/__init__.py

```diff
@@ -30,11 +25,12 @@ def run_pipeline(
         pipeline_id = pipeline_id_or_schema.id
     else:
         raise Exception(
-            "Must either pass a pipeline id, or a pipeline get schema. Not object of type %s in arg 1."
-            % str(pipeline_id_or_schema)
+            "Must either pass a pipeline id, or a pipeline get schema.",
+            "Not object of type %s in arg 1." % str(pipeline_id_or_schema),
```

Comment:
The comma here is new, so the exception message is now a tuple. Is that intentional?

Reply:
no, will fix all the similar ones

Reply:
Thanks! Be aware that the concatenation will not add spaces so you'll have to add them by hand

```python
>>> (
...    "abc"
...    "xyz"
... )
'abcxyz'
```